src/game/GameRow.vue

```diff
@@ -45,7 +48,10 @@ watch(() => row?.shake, value => {
 <style scoped>
 .row {
   position: relative;
-  display: flex;
+  display: grid;
+  grid-template-columns: repeat(5, minmax(20px, 1fr));
+  width: clamp(260px, min(95vw, 50vh), 515px);
```

Comment:
> The reason is that this tries to keep the height in mind, some landscape devices.

Good catch! I have pushed a fix for this along with another fix for iOS. 

The problem with setting each tile's width individually is that you have to do manual calculations or testing on all viewport widths/heights. Instead I elected to set a width on each row (or one could do it on the board itself). This essentially provides the same scaling, but at a macro level.